src/views/route-line/RouteLineView.tsx

```
@@ -1,14 +1,14 @@
+import { API_HOST, API_PATH_JSON_ROUTE } from "../../utils/ApiConstants";
 
-import Main from "../../components/Main.jsx";
-import Nav from "../../components/Nav.jsx";
-import Spinner from "../../components/Spinner.jsx";
-import Error from "../../components/Error.jsx";
-import StopLines from "../../components/StopLines.jsx";
-import { useI18n } from "../../contexts/I18nContext.jsx";
+import Main from "../../components/Main";
+import Nav from "../../components/Nav";
+import Spinner from "../../components/Spinner";
+import Error from "../../components/Error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)